ci/check_format.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
What about using `pre-commit run -a`. This generates a much nicer output with the same effect.

Reply:
Good point, didn't know about this. Will remove `check_format.sh` in favor of `pre-commit run -a`. This will also help with synchronization between the files